DevCheck.ps1

```diff
@@ -605,7 +605,7 @@ function Repair-TAEFService
 
     $root = Get-ProjectRoot
     $cpu = Get-CpuArchitecture
-    $taef = 'Microsoft.Taef.10.58.210222006-develop'
+    $taef = 'Microsoft.Taef.10.75.221207001'
```

Comment:
Why not keep this value in Version.Details.xml?

Reply:
Time.

This is just a quick'n'dirty patch to unblock folks. Another (more complex) PR will use Version.Dependencies.xml, as you'd expect

Patience please. Rome wasn't built in a fortnight :P